### PR TITLE
Feature/e UI 3220 spinner component aa

### PR DESCRIPTION
**Pull request by adnanakgun**
Resolves https://tools.hmcts.net/jira/browse/EUI-3220

Created the component

Notes:
*
*
*

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
LGTMa

File: projects/exui-common-lib/src/lib/components/loading-spinner/loading-spinner.component.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          position: absolute;
+          top: 3px;
+          left: 29px;
+          width: 5px;
+          height: 14px;
+          border-radius: 20%;
+          background: #0b0c0c;
+        }
+
+        &:nth-child(1) {
```
Comment (paul-graham) [inline projects/exui-common-lib/src/lib/components/loading-spinner/loading-spinner.component.scss:45]: Maybe replace all of these with `@include spinnerDivs(12)`, where the mixin looks something like:
```
@mixin spinnerDivs($count) {
  @for $i from 1 through $count {
    &.nth-child(#{$i}) {
      transform: rotate(#{30 * ($i - 1)}deg);
      animation-delay: #{($i / 10) - 1.2}s;
    }
  }
}
```

File: projects/exui-common-lib/src/lib/components/loading-spinner/loading-spinner.component.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<div class="loading-spinner__overlay" [ngStyle]="{'z-index': zIndex}">
+  <div class="loading-spinner__box" [ngStyle]="{'z-index': zIndex + 1}">
+    <div class="loading-spinner__fan">
+      <div></div>
```
Comment (paulhowes-HMCTS) [inline projects/exui-common-lib/src/lib/components/loading-spinner/loading-spinner.component.html:4]: Could reduce this by replacing all the empty divs with <div *ngFor="let i of [].constructor(12)"></div>

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
I think both my recommendation and Paul G's would make the code more efficient.

**Review by paulhowes-HMCTS — APPROVED** [review]

**Review by paul-graham — APPROVED** [review]

**Pull request merged**